updates/rpc_nodes/v6.json

```diff
@@ -0,0 +1 @@
ctive":1,"blockchain":"ARBITRUM_ONE"},{"name":"arbitrum one ankr","endpoint":"https://rpc.ankr.com/arbitrum","weight":0.1,"owned":0,"active":1,"blockchain":"ARBITRUM_ONE"},{"name":"LlamaNodes","endpoint":"https://arbitrum.llamarpc.com","weight":0.3,"owned":0,"active":1,"blockchain":"ARBITRUM_ONE"},{"name":"arbitrum one BlockPi","endpoint":"https://arbitrum.blockpi.network/v1/rpc/public","weight":0.2,"owned":0,"active":1,"blockchain":"ARBITRUM_ONE"},{"name":"arbitrum one PublicNode","endpoint":"https://arbitrum-one-rpc.publicnode.com","weight":0.2,"owned":0,"active":1,"blockchain":"ARBITRUM_ONE"},{"name":"arbitrum one 1rpc","endpoint":"https://1rpc.io/arb","weight":0.1,"owned":0,"active":1,"blockchain":"ARBITRUM_ONE"},{"name":"meowrpc","endpoint":"https://arbitrum.meowrpc.com","weight":0.05,"owned":0,"active":1,"blockchain":"ARBITRUM_ONE"},{"name":"base etherscan","endpoint":"","weight":0.05,"owned":0,"active":1,"blockchain":"BASE"},{"name":"base ankr","endpoint":"https://rpc.ankr.com/base","weight":0.15,"owned":0,"active":1,"blockchain":"BASE"},{"name":"PublicNode","endpoint":"https://base-rpc.publicnode.com","weight":0.2,"owned":0,"active":1,"blockchain":"BASE"},{"name":"base BlockPi","endpoint":"https://base.blockpi.network/v1/rpc/public","weight":0.2,"owned":0,"active":1,"blockchain":"BASE"},{"name":"base 1rpc","endpoint":"https://1rpc.io/base","weight":0.1,"owned":0,"active":1,"blockchain":"BASE"},{"name":"LlamaNodes","endpoint":"https://base.llamarpc.com","weight":0.3,"owned":0,"active":1,"blockchain":"BASE"},{"name":"gnosis etherscan","endpoint":"","weight":0.05,"owned":0,"active":1,"blockchain":"GNOSIS"},{"name":"gnosis ankr","endpoint":"https://rpc.ankr.com/gnosis","weight":0.2,"owned":0,"active":1,"blockchain":"GNOSIS"},{"name":"gnosis BlockPi","endpoint":"https://gnosis.blockpi.network/v1/rpc/public","weight":0.2,"owned":0,"active":1,"blockchain":"GNOSIS"},{"name":"gnosis PublicNode","endpoint":"https://gnosis-rpc.publicnode.com","weight":0.2,"owned":0,"active":1,"blockchain":"GNOSIS"},{"name":"gnosis 1rpc","endpoint":"https://1rpc.io/gnosis","weight":0.2,"owned":0,"active":1,"blockchain":"GNOSIS"},{"name":"gnosischain","endpoint":"https://rpc.gnosischain.com","weight":0.15,"owned":0,"active":1,"blockchain":"GNOSIS"},{"name":"scroll etherscan","endpoint":"","weight":0.05,"owned":0,"active":1,"blockchain":"SCROLL"},{"name":"scroll official","endpoint":"https://rpc.scroll.io/","weight":0.3,"owned":0,"active":1,"blockchain":"SCROLL"},{"name":"scroll ankr","endpoint":"https://rpc.ankr.com/scroll","weight":0.2,"owned":0,"active":1,"blockchain":"SCROLL"},{"name":"scroll blockpi","endpoint":"https://scroll.blockpi.network/v1/rpc/public","weight":0.2,"owned":0,"active":1,"blockchain":"SCROLL"},{"name":"scroll drpc","endpoint":"https://scroll.drpc.org","weight":0.1,"owned":0,"active":1,"blockchain":"SCROLL"},{"name":"chainstacklabs","endpoint":"https://scroll-mainnet.chainstacklabs.com","weight":0.15,"owned":0,"active":1,"blockchain":"SCROLL"}]}
```

Comment:
should we add a skipped test to ping them and test that we are not getting 404? just to check that URLs are correct